WcaOnRails/app/views/chore_mailer/notify_wst_of_assignee.html.erb

```diff
@@ -0,0 +1,35 @@
+Hi WST,
+
+<p>
+  <%= @assignee.name %> has been picked at random to send the next monthly digest on behalf of the WST.
+  <br/>
+  @<%= @assignee.name %>: if for some reason you cannot deal with this task this month, please notify your fellow WST members so that someone can cover for you.
+</p>
+
+<p>
+  The monthly digest should contain a summary of the WST work for the past month.
+  It doesn't need to go into every details, but should definitely mention important changes.
+  You can definitely group similar changes together, for instance by having only one bullet point for packages and locales updates, since we're having dozen of them every month.
+  If you need some inspiration for writing your email, you can check one of the several monthly digests sent in the past.
```

Comment:
Should we instead add a link to a template one?

Reply:
I've added a link to an existing one in our gsuite groups!